README.md

```diff
@@ -39,8 +39,16 @@ And review the following guides:
 * [Configuring Remote Video Subscription](guides/configuring_remote_video_subscriptions.md)
 
 ## Include Amazon Chime SDK in Your Project
+You can download and integrate the Amazon Chime iOS SDKs to your project from both CocoaPods and binaries from Github release.
 
-To include the SDK binaries in your own project, follow these steps.
+### From CocoaPods
+To obtain Amazon Chime iOS SDKs from CocoaPods, add 
```

Comment:
Amazon Chime iOS SDKs -> Amazon Chime SDK

Reply:
This section should also be a "If else" on whether builder needs bitcode or not.
Can you reference [this](https://github.com/aws-amplify/aws-sdk-ios#cocoapods) to provide a detailed guide?